### ISSUE TITLE
Silent mongo shell

### ISSUE THREAD

**Issue by RoxanaTapia** [bbonnin/zeppelin-mongodb-interpreter#21]
Hi @bbonnin 

I got a little problem in my [run command for http get request](https://github.com/bbonnin/zeppelin-mongodb-interpreter/issues/20)

The shell prints every request done as `[main] shell: started program (sh6082):` and shows the command executed with the credentials info in it!

Is there any property [(as listed here)](https://github.com/bbonnin/zeppelin-mongodb-interpreter#configuration) that I can configure to mute mongo shell in my notebook?

I have tried zeppelin `conf/log4j.properties` as recommended [here](https://stackoverflow.com/questions/27781187/how-to-stop-messages-displaying-on-spark-console) without success.

I have also tried [db.setLogLevel](https://docs.mongodb.com/manual/reference/method/db.setLogLevel/) but I'm unauthorized for changing the entire db logging level. 

As you can see in the picture, the shell logs is shown together with my dashboard.

Thanks in advance!

![screen shot 2018-04-16 at 1 31 04 pm](https://user-images.githubusercontent.com/8585764/38806715-7be70eac-417a-11e8-89f9-39070846de61.png)

**Comment by RoxanaTapia** [comment]
I found this lines in the interpreter
```
CommandLine cmdLine = CommandLine.parse(getProperty("mongo.shell.path"))
cmdLine.addArgument("--quiet", false)
```
how can this value be changed?

**Comment by bbonnin** [comment]
unfortunately, there is no way at the moment to change this value.
Feel free to clone the project and modify the code.
I will be more than happy to accept a pull request about this case :-)

**Comment by RoxanaTapia** [comment]
Hi @bbonnin 

Thanks for the opportunity. I created a new mongo shell with the --quiet option included and I was running the interpreter under the mongo path of this new mongo. It didn't solve the issue, so probably the pull request won't help much. 

There is something wrong with this undocumented mongo run() method that executes shell commands. It makes the task of simple HTTP requests very hard to solve. I think if third-party libraries would be included in the mongo interpreter (such as XMLHttpRequest) this problem could be solved easily. 


SOLVED:
I created a .js file on my server with the mongo run() script in it, then in a separated notebook/paragraph I load the script using the mongo native method load(), then in the necessary paragraphs the results can be retrieved using the mongo native method cat() and then data can be processed and no logs are shown in the main notebooks.
